pkg/crc/machine/bundle/metadata.go

```diff
@@ -237,6 +237,13 @@ func GetBundleNameWithoutExtension(bundleName string) string {
 	return strings.TrimSuffix(bundleName, bundleExtension)
 }
 
+func GetBundleNameWithExtension(bundleName string) string {
```

Comment:
Terminology note for later: Ideally it would be possible to use `BundleName` instead of `BundleNameWithoutExtension` and `BundleFilename` instead of `BundleNameWithExtension` but this would require some extensive review of the code base to make it all consistent.